plugins/kubernetes/app/models/watchers/cluster_pod_error_watcher.rb

```diff
@@ -2,34 +2,38 @@ module Watchers
   class ClusterPodErrorWatcher < BaseClusterWatcher
     include Celluloid::Notifications
 
```

Comment:
Nevermind what I said, was not understanding what you meant with calling it directly, I thought you were thinking that I was monkey patching the watchers.

I agree. Including the module was not the best approach in this case. I changed it to call the method directly. Thanks for the suggestion.